code/drasil-example/glassbr/lib/Drasil/GlassBR/DataDefs.hs

```diff
@@ -37,7 +37,8 @@ hFromtQD :: SimpleQDef
 hFromtQD = mkQuantDef minThick hFromtEq
 
 hFromt :: DataDefinition
-hFromt = ddE hFromtQD [dRef astm2009] Nothing "minThick" [hMin]
+hFromt = ddE hFromtQD [dRef astm2009] Nothing "minThick"
+  [hMin, S "nominal thickness t is in" +:+ eS (mkSet Rational (map dbl nominalThicknesses)) ]
```

Comment:
Ah, you moved it here! I'm happy with this change. Keeping it in this PR is fine with me, not worth the effort to remove it. https://github.com/JacquesCarette/Drasil/issues/4084 was originally made about this issue.